doc/api/crypto.md

```diff
@@ -407,12 +407,17 @@ changes:
     description: This method now returns a reference to `decipher`.
 -->
 - `buffer` {Buffer | TypedArray | DataView}
-- Returns: {Cipher} for method chaining.
+- `options` {Object}
```

Comment:
It seems `plaintextLength` should be documented here as well as a property of `options`, in a nested level.

Reply:
Yeah; I had originally planned on writing an `AADOptions` block but it was so short I didn't feel the need to.

Also I added CipherGCMOptions and CipherCCMOptions in @types/node; though not sure how I should go about that in @nodejs/node.

Reply:
If you check other documentations, you'll see how options are handled there. Normally it just get's a short description what it stands for right after the type.